src/utils/api.js

```diff
@@ -18,4 +18,10 @@ export const getCommentsByArticleId = (article_id) => {
     return app.get(`articles/${article_id}/comments`).then((response) => {
         return response.data.comments
     })
-}
\ No newline at end of file
+}
+
+export const voteOnArticle = (article_id, vote) => {
+    return app.patch(`articles/${article_id}`, {inc_votes: vote}).then((response) => {
```

Comment:
should have a function to patch votes imported in from API.js - better to have all interactions with your API as functions within your API.js.